app/requirements.txt

```diff
@@ -1,2 +1,5 @@
-requests==2.28.2
-urllib3<2
\ No newline at end of file
+setuptools>=65.5.1
+requests>=2.20.0
+urllib3<2
+boto3
+globus_sdk
```

Comment:
Could you pin a version of globus-sdk? (We want to be broad in our requirements for the SDK but the backend can/should be as specific as possible)

Also, how was this working before we had globus_sdk as a requirement!?!

Reply:
there was nothing to "not work" before, since the tests were just added :)